roachtest: sqlalchemy failed

[(roachtest).sqlalchemy failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2041142&tab=buildLog) on [release-19.2@6318430c8f0ca0cafb9a34828c9ccdd7ecff68e1](https://github.com/cockroachdb/cockroach/commits/6318430c8f0ca0cafb9a34828c9ccdd7ecff68e1):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/sqlalchemy/run_1
	orm_helpers.go:214,orm_helpers.go:144,sqlalchemy.go:224,sqlalchemy.go:235,test_runner.go:753: 
		Tests run on Cockroach v19.2.7-45-g6318430
		Tests run against sqlalchemy rel_1_3_18
		222 Total Tests Run
		220 tests passed
		2 tests failed
		102 tests skipped
		1 test ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/dialect/test_suite.py::ExceptionTest_cockroachdb+psycopg2_9_5_0::test_integrity_error (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log

```

<details><summary>More</summary><p>


Artifacts: [/sqlalchemy](https://teamcity.cockroachdb.com/viewLog.html?buildId=2041142&tab=artifacts#/sqlalchemy)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Asqlalchemy.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/sqlalchemy.go
+++ b/pkg/cmd/roachtest/sqlalchemy.go
@@ -19,7 +19,7 @@ import (
 	"strings"
 )
 
-var sqlAlchemyResultRegex = regexp.MustCompile(`^(?P<test>test.*::.*::[^ \[\]]*(?:\[.*])?) (?P<result>.*)$`)
+var sqlAlchemyResultRegex = regexp.MustCompile(`^(?P<test>test.*::.*::[^ \[\]]*(?:\[.*])?) (?P<result>\w+)\s+\[.+]$`)
 var sqlAlchemyReleaseTagRegex = regexp.MustCompile(`^rel_(?P<major>\d+)_(?P<minor>\d+)_(?P<point>\d+)$`)
 
 var supportedSQLAlchemyTag = "rel_1_3_17"
@@ -165,7 +165,7 @@ func registerSQLAlchemy(r *testRegistry) {
 		// Note that this is expected to return an error, since the test suite
 		// will fail. And it is safe to swallow it here.
 		rawResults, _ := c.RunWithBuffer(ctx, t.l, node,
-			`cd /mnt/data1/sqlalchemy/ && pytest -s --maxfail=0 `+
+			`cd /mnt/data1/sqlalchemy/ && pytest --maxfail=0 `+
 				`--requirements=cockroachdb.sqlalchemy.test_requirements:Requirements `+
 				`--dburi=cockroachdb://root@localhost:26257/defaultdb?sslmode=disable `+
 				`test/dialect/test_suite.py`)
@@ -222,7 +222,7 @@ func registerSQLAlchemy(r *testRegistry) {
 		}
 
 		results.summarizeAll(
-			t, "sqlalchemy" /* ormName */, blocklistName, expectedFailures, version, latestTag)
+			t, "sqlalchemy" /* ormName */, blocklistName, expectedFailures, version, supportedSQLAlchemyTag)
 	}
 
 	r.Add(testSpec{

